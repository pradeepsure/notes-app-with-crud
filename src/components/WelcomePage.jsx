import React, { Fragment } from 'react';
import { MDBView, MDBMask } from "mdbreact";


import takeANote from '../images/takenote.jpg';


function WelcomePage(props) {
    return (
        <Fragment>
            
                            <MDBView  >
                                <img className="d-block w-50" src={takeANote} alt="First slide" />                                
                                <MDBMask overlay="black-light" />
                            </MDBView>
                        
        </Fragment>
    );


};

export default WelcomePage;