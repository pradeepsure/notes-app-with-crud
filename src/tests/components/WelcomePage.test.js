import WelcomePage from '../../components/WelcomePage';
import React from 'react';
import { shallow, mount } from 'enzyme';
import { createSerializer } from 'enzyme-to-json';

expect.addSnapshotSerializer(createSerializer({mode: 'deep'}));


test('WelcomePage component testing!!!', () => {    
    const wrapper =shallow(<WelcomePage/>);
    expect(wrapper).toMatchSnapshot();
});