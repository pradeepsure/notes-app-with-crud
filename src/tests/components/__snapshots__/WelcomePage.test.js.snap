// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WelcomePage component testing!!! 1`] = `
<Fragment>
  <t
    cascade={false}
    className=""
    hover={false}
    rounded={false}
    src=""
    tag="div"
    waves={false}
    zoom={false}
  >
    <img
      alt="First slide"
      className="d-block w-50"
      src="takenote.jpg"
    />
    <t
      className=""
      overlay="black-light"
      pattern=""
      tag="div"
    />
  </t>
</Fragment>
`;
